### DISCUSSION TITLE
middle scroll

### DISCUSSION THREAD

**Discussion by Tetuas** [KadeArchive/Kade-Engine#1045] [Q&A]
Will there be a middle scroll option soon?

**Comment by guizinhomartinez** [comment]
no i guess

**Comment by NoiceBroice** [comment]
if they do I hope they call it a centered strumline. because middle scroll is misleading, it sounds like im gonna be playing taiko

**Reply by isakube** [reply]
that's sidescroll

**Comment by Lucky-56** [comment]
isn't the optimized option middle scroll ?

**Reply by ACardboardBox1** [reply]
> 
> 
> isn't the optimized option middle scroll ?

shit ur right

**Reply by Tetuas** [reply]
the what

**Reply by KlutchDj** [reply]
optimization Mode , its in the kade settings.

**Reply by Tetuas** [reply]
where

**Reply by Tetuas** [reply]
got it

**Reply by SuperSpinDashGamer39** [reply]
hey in what hx file does optimization work
